src/stability_sdk/client.py

```diff
@@ -268,12 +268,19 @@ def __init__(self, channel: Optional[grpc.Channel]=None, stub: Optional[generati
         self._asset = ApiEndpoint(stub, 'asset-service')
         self._generate = ApiEndpoint(stub, 'stable-diffusion-v1-5')
         self._inpaint = ApiEndpoint(stub, 'stable-inpainting-512-v2-0')
-        self._interpolate = ApiEndpoint(stub, 'interpolate-v1')
+        self._interpolate = ApiEndpoint(stub, 'interpolation-server-v1')
         self._transform = ApiEndpoint(stub, 'transform-server-v1')
         self._debug_no_chains = False
         self._max_retries = 3 # retry request on RPC error
         self._retry_obfuscation = False # retry request with different seed on classifier obfuscation
 
+        logger.warning(
+            "\n"
+            "The functionality available through this Api is in beta and subject to changes in both functionality and pricing.\n"
```

Comment:
Unless we're standardizing on `Api` spelling, it's likely better to say `API` here.

Reply:
Thanks! I changed it so it's clear it's referring to the "Api" class that is being instantiated. Need to think of a better name for it 🤔